pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -443,6 +443,14 @@ def wbem_server(self):
                 verbose=ctx.obj.verbose)
         return self._wbem_server
 
+    @property
+    def connected(self):
+        """
+        Current connected state. Returns True if connected and
+        False if not connected.
+        """
+        return self._wbem_server
```

Comment:
This does not return a boolean but the PywbemServer object. Change to using `bool()` around it.

Reply:
DONE